README.md

```diff
@@ -3,7 +3,7 @@
 Corepack is a zero-runtime-dependency Node.js script that acts as a bridge
 between Node.js projects and the package managers they are intended to be used
 with during development. In practical terms, **Corepack lets you use Yarn, npm,
-and pnpm without having to install them**.
+and other supported package managers without having to install them**.
```

Comment:
Why did we remove pnpm from this list?

Reply:
Mistake; I meant to remove npm and leave pnpm.